tests/integration_tests/data_steward/validation/ehr_union_test.py

```diff
@@ -63,7 +62,7 @@ def setUp(self):
 
     def _empty_hpo_buckets(self):
```

Comment:
oh wait.  This is in a loop... hmm

Reply:
well, it can still be shorten:
```
def _empty_hpo_buckets(self):
        for hpo_id in self.hpo_ids:
            self.storage_client.empty_bucket(hpo_id)
```

upto you

Reply:
`self.storage_client.empty_bucket(hpo_id)` wouldn't work because `empty_bucket(...)` takes a Bucket object or Bucket name as an argument and hpo_id is an id.